lib/helper.js

```diff
@@ -135,6 +135,23 @@ PokerHelper.prototype.nextActivePlayer = function(lineup, playerPos) {
   return newPos;
 }
 
+PokerHelper.prototype.findMaxRaise = function(lineup, dealer, lastRoundMaxBet) { 
```

Comment:
then it would a bet and it would be the big blind. Or the your stacksize if you dont have the big blind.